utils.sh

```diff
@@ -35,6 +35,25 @@ function local_gateway_mode(){
   cp assets/templates/cluster-network-00-gateway-mode.yaml.optional assets/generated/cluster-network-00-gateway-mode.yaml
 }
 
+function ensure_line() {
+  file=$1
+  line=$2
+
+  grep -q "$line" $file || echo "$line" | sudo tee -a $file
+}
+
+function configure_chronyd() {
+  sudo dnf install chrony && sudo systemctl enable chronyd
+
+   ensure_line /etc/chrony.conf "allow ${EXTERNAL_SUBNET_V4}"
+   ensure_line /etc/chrony.conf "allow ${EXTERNAL_SUBNET_V6}"
+   ensure_line /etc/chrony.conf "allow ${PROVISIONING_NETWORK}"
+
+  sudo systemctl restart chronyd
+  sudo firewall-cmd --permanent --add-service=ntp
```

Comment:
I think this will apply to the default e.g public zone, but I think it should be enough to enable this internally for the libvirt zone? (at least when `MANAGE_BR_BRIDGE` is specified)

Reply:
You are right, and this is why this isn't working :-)

Without --zone=libvirt
```
sh-4.4# chronyd -q 'server fd2e:6f44:5dd8:c956::1 iburst'
2021-07-08T11:16:49Z chronyd version 3.5 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP +SCFILTER +SIGND +ASYNCDNS +SECHASH +IPV6 +DEBUG)
2021-07-08T11:17:00Z No suitable source for synchronisation
```

With
```
sh-4.4# chronyd -q 'server fd2e:6f44:5dd8:c956::1 iburst'
2021-07-08T11:17:08Z chronyd version 3.5 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP +SCFILTER +SIGND +ASYNCDNS +SECHASH +IPV6 +DEBUG)
2021-07-08T11:17:12Z System clock wrong by 7.263145 seconds (step)
2021-07-08T11:17:19Z chronyd exiting
```